modules/core/src/v2/trading/tradingPartners.ts

```diff
@@ -80,4 +96,31 @@ export class TradingPartners {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Refer a new trading partner to join the BitGo network and become your partner.
+   * @param params
+   * @param params.institutionName name of the institution to invite
+   * @param params.contactName full name of a member of the institution to contact
+   * @param params.contactEmail email address of the contact
+   * @param params.contactPhoneNumber phone number of the contact
+   * @param params.memo memo to send to the trading partner when sending the invite
+   * @param callback
+   */
+  referByCode(params: TradingPartnerReferByCodeParameters, callback?: NodeCallback<{}>): Bluebird<{}> {
```

Comment:
Can you add a return type? Same for the method `refer` above

Reply:
as discussed, fixed for referByCode but not for refer